scimma_admin/hopskotch_auth/views.py

```diff
@@ -58,6 +58,10 @@ def logout(request):
     return HttpResponse("you're logged out!")
 
 
+def login_failure(request):
+    return render(request, 'hopskotch_auth/login_failure.html')
```

Comment:
Is there any message we can show here?

I think we should get some query parameters, laid out here: https://openid.netcspecs/openid-connect-core-1_0.html#AuthError

At a minimum, presenting the user with the request's error code (which might come from `request.GET.get("error", "unknown-code")`) and description (`request.GET.get("error_description", "unknown")`) seems worthwhile. Logging them also seems good.

Reply:
Unfortunately, the way we get to this page is a redirect from inside of `mozilla_django_oidc.views.OIDCAuthenticationCallbackView`, which sends no query parameters, or any other kind of error information I can find in the `request` object we get.

Reply:
huh, you're right: https://github.com/mozilla/mozilla-django-oidc/issues/240